tests/certification/state/redis/docker-compose.yml

```diff
@@ -1,7 +1,6 @@
 version: "3.3"
 services:
   redis:
-    image: 'redislabs/redisearch:latest'
+    image: 'redislabs/redisearch:edge'
```

Comment:
edge includes the later 2.8 redis search module.

Reply:
Please don't pin to non-stable builds. I trust that the bug was there :)

Reply:
Makes sense, done!